mandatory/2-function-creation.js

```diff
@@ -4,7 +4,15 @@
     1. the user should be 18 or older
     2. the user must be logged in
 */
-function isAcceptableUser(userAge, isLoggedIn) {}
+let userAge = 18;
```

Comment:
Hi Christina, well done! :) these excersises were quite the challenge! Just a wee thing, I dont think is necessary here to give a value to userAge and isLoggedIn because the test will be doing that?

Reply:
Hi Irianni! Thank you for your comment.
Yes you are correct! I repeated the mistake I made last week! I will keep it in mind in future!